### DISCUSSION TITLE
Segmentation fault (signal 11)

### DISCUSSION THREAD

**Discussion by AIMOMO1314** [su2code/SU2#2310] [Q&A]
Hi,
Due to my special needs, I made some modifications to the Lax method in SU2-NEMO.
I was able to compile it successfully after the changes were made, but there was a segment fault when I ran it.
I carefully reviewed my code but did not find the problem, can someone help me to solve this problem？
If necessary, I can attach the code I modified

**Comment by bigfooted** [comment]
Which version of Lax' scheme are you implementing?
if you compile SU2 with debugging support and run it in debug mode in vscode, you usually get the exact line where the segmentation fault occurs. That is the first thing to do if you have not done so already.

**Reply by AIMOMO1314** [reply]
Version 8.0.1 
Thank you very much for your suggestions. I'll give it a try
